packages/bot/src/scratch/utils/contracts-for.js

```diff
@@ -0,0 +1,369 @@
+import PendingPromise from './pending-promise';
```

Comment:
@aaron-binary i think its better if we move api to services. i dont think it should be here

Reply:
and also dont call it utils. they are not utilities.